templates/logrotate.j2

```diff
@@ -0,0 +1,16 @@
+/var/log/pgbouncer/instance_*/pgbouncer.log {
+    rotate 10
+    missingok
+    sharedscripts
+    notifempty
+    nocompress
+    daily
+    create 0600 postgres postgres
+    dateext
+    dateformat -%Y%m%d_%H:%M.log
+    postrotate
+    {% for id in service_ids %}
+         pebble restart pgbouncer_{{ id }}
+    {% endfor %}
```

Comment:
I don't think pebble can reload.

Reply:
Does `pebble signal` help on this?

Reply:
It does. Thanks for the suggestion.

Reply:
Q: why not `/bin/kill -HUP ${pgbouncer_pid}` ?